source/_patterns/01-molecules/27-styled-ibetype-icons/styled-ibetype-icons.md

```diff
@@ -0,0 +1,4 @@
+---
+title: Styled IBE Type Icons
```

Comment:
Is it possible to call it "IBE Type Icons"?

Reply:
I called it like this because of the names on the tickets, but it doesn't make sense to have the _Styled_ word in a Styleguide project :D
I'll remove it, thanks!